api/wikis/serializers.py

```diff
@@ -99,6 +108,41 @@ class NodeWikiSerializer(WikiSerializer):
         related_view_kwargs={'wiki_id': '<_id>'},
     )
 
+    def update(self, instance, validated_data):
+        auth = Auth(self.context['request'].user)
+        new_page_name = validated_data.get('page_name')
+        validated_data.pop('content', None)
+
+        if new_page_name:
```

Comment:
Is the check necessary?

Reply:
Nope! I think this was from before I added `allow_blank=False` to `name`. Will remove.